app/controllers/photos_controller.rb

```diff
@@ -1,26 +1,30 @@
+# frozen_string_literal: true
+
 class PhotosController < ApplicationController
   def index
     @photo = Photo.new
     search_image
   end
 
   def create
-    @photo = Photo.new(photo_params)
-    @photo.save
+    @photo = current_user.microposts.build(photo_params)
+    if @photo.save
+      flash[:success] = "Picture has been added!"
   end
 
-def favorite
-  @photos = Photo.all
-end
+  def favorite
+    @photos = current_user.Photo.all
+  end
 
-def destroy
-  @photo = Photo.find(params[:id])
-  @photo.destroy
-end
+  def destroy
+    @photo = Photo.find(params[:id])
```

Comment:
phải check photo có phải của current_user không nữa

```suggestion
    @photo =current_user.photos.find(params[:id])
```